samples/azure-communication-chat-samples/src/androidTest/java/com/azure/azure_communication_chat_samples/CreateChatThread.java

```diff
@@ -0,0 +1,106 @@
+package com.azure.azure_communication_chat_samples;
+
+import com.azure.android.communication.chat.BuildConfig;
+import com.azure.android.communication.chat.ChatAsyncClient;
+import com.azure.android.communication.chat.ChatClientBuilder;
+import com.azure.android.communication.chat.models.ChatParticipant;
+import com.azure.android.communication.chat.models.CreateChatThreadOptions;
+import com.azure.android.communication.chat.models.CreateChatThreadResult;
+import com.azure.android.communication.common.CommunicationUserIdentifier;
+import com.azure.android.core.http.HttpClient;
+import com.azure.android.core.http.HttpRequest;
+import com.azure.android.core.http.okhttp.OkHttpAsyncClientProvider;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+import java9.util.concurrent.CompletableFuture;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+public class CreateChatThread {
+    protected static final String ENDPOINT = BuildConfig.COMMUNICATION_SERVICE_ENDPOINT;
+    protected static final String ACCESS_KEY = BuildConfig.COMMUNICATION_SERVICE_ACCESS_KEY;
+    protected static final String THREAD_MEMBER_1 = BuildConfig.COMMUNICATION_CHAT_THREAD_MEMBER_1;
+    protected static final String THREAD_MEMBER_2 = BuildConfig.COMMUNICATION_CHAT_THREAD_MEMBER_2;
+    private static CommunicationUserIdentifier secondThreadMember;
+    private static CommunicationUserIdentifier firstThreadMember;
+
+    private static ChatAsyncClient client;
+
+    private static ChatClientBuilder getChatClientBuilder() {
+        HttpClient httpClient = new OkHttpAsyncClientProvider().createInstance();
+        ChatClientBuilder builder = new ChatClientBuilder();
+
+        builder
+            .endpoint(ENDPOINT)
+            .httpClient(httpClient);
+
+        builder.credentialPolicy(chain -> {
+            HttpRequest request = chain.getRequest();
+            request.getHeaders().put("Authorization", "Bearer " + ACCESS_KEY);
+            chain.processNextPolicy(request);
+        });
+
+        return builder;
```

Comment:
We can probably simplify this but I'm not 100% sure to what degree it would be better for readability.

```suggestion
        ChatClientBuilder builder = new ChatClientBuilder()
            .endpoint(ENDPOINT)
            .httpClient(httpClient)
            .credentialPolicy(chain -> {
                HttpRequest request = chain.getRequest();
                request.getHeaders().put("Authorization", "Bearer " + ACCESS_KEY);
                chain.processNextPolicy(request);
            });

        return builder;
```